translate/cloud-client/translate_v3_detect_language.py

```diff
@@ -16,7 +16,7 @@
 from google.cloud import translate
 
 
-def sample_detect_language(project_id="YOUR_PROJECT_ID"):
+def detect_language(project_id="YOUR_PROJECT_ID"):
```

Comment:
Here and there

Why do you want to change this? It's not a big deal, but this will cause a discrepancy between here and our documentation site, maybe for a short period of time.

If you are aware of the consequences of the change, and you still want to change this for some reason, that's OK though.

Reply:
These samples were generated by GAPIC initially, and at first all generated methods have `sample_` prefix. Then, I have refactored a bit according to reviewers requests. Along the way, I may have forgotten to remove the prefixes and change updated method names in the tests. Now, keep them all method names consistent, and I renamed them.

Reply:
alright